$NetBSD: patch-ac,v 1.4 2006/09/07 08:46:20 wennmach Exp $

Do not install the des executable, nor the des.1 man page. This avoids
a conflict with the openssl package, which installs an executable with
the same name (and functionality) into the same place.

--- lib/des/Makefile.am.orig	Mon Apr  1 11:48:01 2002
+++ lib/des/Makefile.am	Mon Apr  1 11:49:39 2002
@@ -13,7 +13,7 @@
 
 TESTS = destest mdtest md5crypt_test
 CHECK_LOCAL = 
-bin_PROGRAMS = des #rpw speed
+bin_PROGRAMS =
 
 des_SOURCES = des.c des_ver.h
 
@@ -59,7 +59,7 @@
 EXTRA_libdes_la_SOURCES = dllmain.c passwd_dialog.aps passwd_dialog.clw \
 	passwd_dialog.rc passwd_dialog.res passwd_dlg.c passwd_dlg.h resource.h
 
-man_MANS = des.1 des_crypt.3
+man_MANS = des_crypt.3
 
 ## this is an awful lot of junk, but it's just as well to include everything
 EXTRA_DIST =		\
